### ISSUE TITLE
Seems a bug

### ISSUE THREAD

**Issue by xingzheduan** [viblo/pymunk#122]
Something wrong occured when I tested a example from the examples folder.Error comes from the examples/breakout.py. I used python2.7.12, version of pymunk is 5.3.1  my  system is ubuntu16.04 LTS.
Here is the traceback:
`From cffi callback <function cf at 0x7f29ca299d70>:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pymunk/collision_handler.py", line 107, in cf
    x = func(Arbiter(_arb, self._space), self._space, self._data)
  File "breakout.py", line 140, in pre_solve
    arbiter.contact_point_set = set_        
  File "/usr/local/lib/python2.7/dist-packages/pymunk/arbiter.py", line 47, in _set_contact_point_set
    _set.normal = point_set.normal
TypeError: initializer for ctype 'cpVect' must be a list or tuple or dict or struct-cdata, not Vec2d
From cffi callback <function cf at 0x7f29ca299d70>:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pymunk/collision_handler.py", line 107, in cf
    x = func(Arbiter(_arb, self._space), self._space, self._data)
  File "breakout.py", line 140, in pre_solve
    arbiter.contact_point_set = set_        
  File "/usr/local/lib/python2.7/dist-packages/pymunk/arbiter.py", line 47, in _set_contact_point_set
    _set.normal = point_set.normal
TypeError: initializer for ctype 'cpVect' must be a list or tuple or dict or struct-cdata, not Vec2d
From cffi callback <function cf at 0x7f29ca299d70>:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pymunk/collision_handler.py", line 107, in cf
    x = func(Arbiter(_arb, self._space), self._space, self._data)
  File "breakout.py", line 140, in pre_solve
    arbiter.contact_point_set = set_        
  File "/usr/local/lib/python2.7/dist-packages/pymunk/arbiter.py", line 47, in _set_contact_point_set
    _set.normal = point_set.normal
TypeError: initializer for ctype 'cpVect' must be a list or tuple or dict or struct-cdata, not Vec2d`

**Comment by cprogrammer1994** [comment]
I was able to reproduce the problem (tested with HEAD = 9ae006f1b656474d96e8ccf4a76db1fb053adacb)

```
user@ubuntu:~/Desktop/pymunk/examples$ uname -a
Linux ubuntu 4.10.0-28-generic #32~16.04.2-Ubuntu SMP Thu Jul 20 10:19:13 UTC 2017 i686 i686 i686 GNU/Linux
user@ubuntu:~/Desktop/pymunk/examples$ python --version
Python 2.7.12
user@ubuntu:~/Desktop/pymunk/examples$ python -m pip freeze
adium-theme-ubuntu==0.3.4
cffi==1.10.0
pycparser==2.18
pygame==1.9.3
pymunk==5.3.1
unity-lens-photos==1.0

```

![image](https://user-images.githubusercontent.com/11232402/30322416-1583494e-97c2-11e7-9ad2-87fe1e4d31bd.png)

```
~/Desktop/pymunk/examples$ python breakout.py 
Loading chipmunk for Linux (32bit) [/home/user/.local/lib/python2.7/site-packages/pymunk/libchipmunk.so]
From cffi callback <function cf at 0xb6150064>:
Traceback (most recent call last):
  File "/home/user/.local/lib/python2.7/site-packages/pymunk/collision_handler.py", line 107, in cf
    x = func(Arbiter(_arb, self._space), self._space, self._data)
  File "breakout.py", line 140, in pre_solve
    arbiter.contact_point_set = set_        
  File "/home/user/.local/lib/python2.7/site-packages/pymunk/arbiter.py", line 47, in _set_contact_point_set
    _set.normal = point_set.normal
TypeError: initializer for ctype 'cpVect' must be a list or tuple or dict or struct-cdata, not Vec2d
From cffi callback <function cf at 0xb6150064>:
Traceback (most recent call last):
  File "/home/user/.local/lib/python2.7/site-packages/pymunk/collision_handler.py", line 107, in cf
    x = func(Arbiter(_arb, self._space), self._space, self._data)
  File "breakout.py", line 140, in pre_solve
    arbiter.contact_point_set = set_        
  File "/home/user/.local/lib/python2.7/site-packages/pymunk/arbiter.py", line 47, in _set_contact_point_set
    _set.normal = point_set.normal
TypeError: initializer for ctype 'cpVect' must be a list or tuple or dict or struct-cdata, not Vec2d
From cffi callback <function cf at 0xb6150064>:
Traceback (most recent call last):
  File "/home/user/.local/lib/python2.7/site-packages/pymunk/collision_handler.py", line 107, in cf
    x = func(Arbiter(_arb, self._space), self._space, self._data)
  File "breakout.py", line 140, in pre_solve
    arbiter.contact_point_set = set_        
  File "/home/user/.local/lib/python2.7/site-packages/pymunk/arbiter.py", line 47, in _set_contact_point_set
    _set.normal = point_set.normal
TypeError: initializer for ctype 'cpVect' must be a list or tuple or dict or struct-cdata, not Vec2d
```

**Comment by viblo** [comment]
Thanks for the report! I have committed a fix to master. Since its a big serious Ill put out a new release as well.

**Comment by viblo** [comment]
Fixed in the newly released 5.3.2 version.
